src/server/pegasus_mutation_duplicator.cpp

```diff
@@ -250,6 +250,9 @@ void pegasus_mutation_duplicator::duplicate(mutation_tuple_set muts, callback cb
             // Because DUPLICATE comes from other clusters should not be forwarded to any other
             // destinations. A DUPLICATE is meant to be targeting only one cluster.
             continue;
+        } else if (rpc_code == dsn::apps::RPC_RRDB_RRDB_BULK_LOAD) {
```

Comment:
It's not a problem to add a judgement here like this, but how to deal with the ingested data? As a "master cluster", should the data been ingested to be duplicate to remote cluster?

Reply:
Currently, there are many users that require both bulkload and duplication.
So the current strategy is that users have to ensure that a bulkload is also performed on the bakcup-cluster.
We will have an external component that can detect the bulkload usage of a cluster's table. If the master-cluster does bulkload but the backup-cluster does not, an email notification will be sent.

Reply:
Last week our cluster crashed for this reason. 
A user did a bulkload on a table with dup enabled, which caused the whole cluster to coredump. We think this is very unsafe.

Reply:
When bulkload on source and target clusters, will you stop client's normal writing? How to keep the consistent of the 2 clusters if not?

Reply:
The current situation will not stop normal writing. And it may make some unconsistent data between 2 clusters.
In general, we will inform users that this is a situation that will lead to data inconsistency between master-cluster and backup-cluster.

Reply:
In other words, if there is only one cluster, if there is both nomral writing and bulkload (without ingest_behind parameter), it may also cause inconsistency between actual data and expected data.

Reply:
So it's better to deny bulkloading on a duplicating table (you can permit this risk in you own way).

This patch's change looks good to me.